react/components/GalleryLayoutRow.tsx

```diff
@@ -52,10 +52,9 @@ const GalleryLayoutRow: React.FC<GalleryLayoutRowProps> = ({
     <>
       {products.map((product, index) => {
         const absoluteProductIndex = rowIndex * itemsPerRow + index + 1
-
         return (
           <div
-            key={product.productId}
+            key={product.cacheId}
```

Comment:
could you test it in a store with the search-resolver@0.x? I think it won't cause any problems but just to make sure 😅